Sources/_Concurrency/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+#[[
+Copyright © 2018 Saleem Abdulrasool <compnerd@compnerd.org>.
+All Rights Reserved.
+
+SPDX-License-Identifier: BSD-3
+#]]
+
+add_library(swiftConcurrencyShims
+  ConcurrencyShims.swift)
+set_target_properties(swiftConcurrencyShims PROPERTIES
+  Swift_MODULE_NAME _Concurrency)
+target_compile_options(swiftConcurrencyShims PRIVATE
+  -parse-stdlib
+  -disallow-use-new-driver
+  "SHELL:-Xfrontend -enable-resilience")
+target_link_libraries(swiftConcurrencyShims PUBLIC
+  swiftCore)
+target_link_options(swiftConcurrencyShims PRIVATE
+  "SHELL:-Xclang-linker -nostdlib")
```

Comment:
I wonder if this is something that we should keep.  We do want the dependency on other language runtimes once the code will be added.

Reply:
I don't feel strongly, but it might be good future proofing. And it matches what we do everywhere else.

Reply:
Fair enough, leaving this for now won't hurt anything.